docs/releases/v2023.1.rst

```diff
@@ -0,0 +1,121 @@
+Gluon 2022.1
+============
+
+Important notes
+---------------
+
+Upgrades to v2022.1 and later releases are only supported from releases v2021.1 and later. This is due to migrations that have been removed to simplify maintenance.
+
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- Extreme Networks
+
+  - WS-AP3805i
+
+ath79-nand
+~~~~~~~~~~
+
+- GL.iNet
+
+  - GL-XE300
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - EAP615-Wall
+
+- Wavlink
+
+  - WS-WN572HP3 4G
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - TL-MR6400 (v5)
+
+
+Features
+--------
+
+DNS-Caching
+~~~~~~~~~~~
+
+DNS-Caching using the dnsmasq resolver is reintroduced. This requires configuring the
+caching parameters. See the :ref:`DNS-Caching documentation <dns-caching>` section for
+details.
+
+Cellular Modem Support
+~~~~~~~~~~~~~~~~~~~~~~
+
+Support for using cellular data connections as the primary uplink connection has been added
+to Gluon. This is supported for nodes which come with a built-in cellular modem. This feature
+requires user-configuration for the APN which can be performed using the Advanced Settings in
```

Comment:
Style: Two sentences starting with "This" in a row, lots of "which". Suggestion:

"The required user configuration for the APN can be performed in the Advanced Settings in Config Mode."